contrib/vmap_extractor/CMakeLists.txt

```diff
@@ -19,6 +19,14 @@ ADD_DEFINITIONS("-Wall")
 ADD_DEFINITIONS("-ggdb")
 ADD_DEFINITIONS("-O3")
 
+if (APPLE)
```

Comment:
This worked before for other apple so this mean it may broke them now. I insist on the "may" word. No idea if it will realy or not.